utils/json-utils/src/main/scala/pl/touk/nussknacker/engine/json/swagger/extractor/JsonToNuStruct.scala

```diff
@@ -90,7 +58,8 @@ object JsonToNuStruct {
             _.asArray,
             _.zipWithIndex.map { case (el, idx) => JsonToNuStruct(el, elementType, s"$path[$idx]") }.asJava
           )
-        case obj: SwaggerObject => extractObject(obj)
+        case obj: SwaggerObject =>
+          JsonTypedMap(json.asObject.get, obj, path)
```

Comment:
It may be a good idea to pass object's size, to preallocate internal HashMap using correct `initialSize` hint